api/Request.json

```diff
@@ -1097,7 +1097,7 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "132"
```

Comment:
This appears to be preview, and is covered by https://github.com/mdn/browser-compat-data/pull/24656